docs/src/main/sphinx/installation/cli.rst

```diff
@@ -80,6 +80,35 @@ The detailed behavior is as follows:
   authentication system configuration.
 * Expired tokens force you to log in again.
 
+.. _cli-certificate-auth:
+
+Certificate authentication
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Use the following command line arguments to allow you to connect the CLI to a
+cluster that enables client :doc:`certificate authentication
+</security/certificate>`.
+
+.. list-table:: Certificate authentication options
```

Comment:
CLI options for certificate authentication

Reply:
OK